Code/EduCopter_Backend/EduCopter.Domain/Geography/Province.cs

```diff
@@ -5,5 +5,7 @@ public class Province : Entity
         public string Name { get; set; }
 
         public Guid CountryId { get; set; }
+
+        public Guid MapId { get; set; }
     }
```

Comment:
Non-nullable property 'Username' must contain a non-null value when exiting constructor. Consider declaring the property as nullable.

Kan je me iets meer informatie verschaffen over deze waarschuwing in build voordat we mergen?

Reply:
Is standaard melding als er geen constructor is en er geen standaard waarde staat bij die property